src/pyOpenMS/CMakeLists.txt

```diff
@@ -631,7 +631,7 @@ if ("${PY_NUM_MODULES}" GREATER "1")
         SUFFIX ${PYTHON_EXT_SUFFIX} # since CMake 3.17 you can add WITH_SOABI to Python_add_library
         CXX_VISIBILITY_PRESET ${VIS_PRESET}
         VISIBILITY_INLINES_HIDDEN ${INLINES_HIDDEN}
-        INSTALL_RPATH ${PY_RPATH}
+        INSTALL_RPATH "${PY_RPATH}"
```

Comment:
so this fails on linux and mac?

Reply:
Probably not. But this needs to be checked.